etc/debian/control

```diff
@@ -0,0 +1,36 @@
+Source: scancode-toolkit
+Section: unknown
+Priority: optional
+Maintainer: root <root@40825ee16090>
+Build-Depends: debhelper (>= 11), dh-python, python, python-setuptools
+Standards-Version: 4.1.3
+Homepage: <insert the upstream URL, if relevant>
+X-Python-Version: >= 2.6
+X-Python3-Version: >= 3.2
+
+Package: python-scancode-toolkit
```

Comment:
Why would this be prefixed with `python` ?

Reply:
Correctly pointed out ! 
Is scancode-toolkit been used as library(python-module) apart from application.
In debian there is a general convention to name python module package as python-module*.